baybe/utils/basic.py

```diff
@@ -181,3 +182,75 @@ def find_subclass(base: type, name_or_abbr: str, /):
             f"The class name or abbreviation '{name_or_abbr}' does not refer to any "
             f"of the subclasses of '{base.__name__}'."
         )
+
+
+def register_hooks(
```

Comment:
since this is really a userfacing utility, should we also enable it via `from bayb.utils import register_hooks` ?